install/etc/services.available/10-mariadb/run

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/with-contenv bash
 
 source /assets/functions/00-container
+# shellcheck disable=SC2034
 PROCESS_NAME="mariadb"
 
 check_container_initialized
 check_service_initialized init
-liftoff
 
 print_info "Starting MariaDB ${MARIADB_VERSION}"
-silent exec /usr/bin/mysqld --defaults-file=/etc/mysql/my.cnf --user=mariadb --console --skip-name-resolve
\ No newline at end of file
+silent exec /usr/bin/mysqld --defaults-file=/etc/mysql/my.cnf --user=mariadb --console --skip-name-resolve
+
+liftoff
```

Comment:
Thanks for catching this. Can you move liftoff statement to before line 10 ? This will allow us to determine that the script completed its routines before starting `mysqld` instead of if the process fails.

Reply:
Yep, check up